cmdLine/src/main/scala/topgun/cmdline/FileParser.scala

```diff
@@ -11,6 +12,22 @@ class FileParser(file: File, cmdLine: JfrParseCommandLine, totals: Totals, confi
 
   import configuration.{includeStack, includeThread}
 
+  def verify(recordingFile: RecordingFile): Boolean = {
+
+    val allEventsTypes = recordingFile.readEventTypes();
+    val eventTypes = allEventsTypes.stream().map(eventType => eventType.getLabel).collect(Collectors.toList[String])
```

Comment:
a set would be better here - we only care about existance we dont need to keep all of them